src/site/content/en/blog/building-multiple-pwas-on-the-same-domain/index.md

```diff
@@ -0,0 +1,264 @@
+  installable experience.
+- **Storage:** Each app has its own caches, local storage, and basically all
+  forms of device-local storage, without sharing them with the others.
+- **Service Workers:** Each app has its own service worker(s) for the registered
+  scopes.
+- **Permissions:** Permissions are also scoped by origins. Thanks to that, users
+  will know exactly for which service they are giving permissions for, and
+  features like notifications will be properly attributed to each app.
+
+Creating such a degree of isolation is the most desirable in the use case of
+multiple independent PWAs, so **we strongly recommend this approach**.
+
+If apps on subdomains want to share local data with each other they will still
+be able to do it via **cookies**, or for more advanced scenarios they could
+consider synchronizing the storage through a server.
+
+<figure class="w-figure">
+  {% Img src="image/26V1DWN36MZr3mUo8ChSBlCpzp43/FfuseK64y4PRqXYEGEt9.png", alt="ALT_TEXT_HERE", width="800", height="421" %}
+  <figcaption class="w-figcaption">Building different PWAs in distinct origins, by using subdomains is a good practice.</figcaption>
+</figure>
+
+## Using the same origin
+
+The second approach is building the different PWAs on the same origin. This
+includes the following scenarios:
+
+### Non-overlapping paths
+
+Multiple PWAs or conceptual "web apps", hosted on the same origin, with
+non-overlapping paths. For example:
+
+- `https://example.com/app1/`
+- `https://example.com/app2/`
+
+### Overlapping/nested paths
+
+Multiple PWAs on the same origin, one of whose scope is nested inside the other:
+
+- `https://example.com/` (the "outer app")
+- `https://example.com/app/` (the "inner app")
+
+The service worker API and manifest format allow you to do either of the above,
+using path-level scoping. However, in both cases, using the same origin presents
+many problems and limitations, the root of which stems from the fact that the
+browser won't fully consider these to be distinct "apps", therefore **this
+approach is discouraged**.
+
+<figure class="w-figure">
+  {% Img src="image/26V1DWN36MZr3mUo8ChSBlCpzp43/6HX5zEb58sEEWQoJt82m.png", alt="ALT_TEXT_HERE", width="800", height="420" %}
+  <figcaption class="w-figcaption">Using paths (overlapping or not) to provide two independent PWAs (“app1”, “app2”) under the same origin is discouraged.</figcaption>
+</figure>
+
+In the next section, we analyze these challenges in more detail, and what can be
+done, if using separate origins is not an option.
+
+## Challenges for multiple, same-origin PWAs
+
+Here are some practical issues common to both "same-origin" approaches:
+
+- **Storage:** Cookies, local storage, and all forms of device-local storage are
+  shared between apps. For that reason, if the user decides to wipe local data
+  for one app, it will wipe all the data from the origin; there's no way of
+  indicating to do this for a "single app". Note that Chrome and some other
```

Comment:
```suggestion
  for one app, it will wipe all the data from the origin; there's no way 
 to do this for a single app. Note that Chrome and some other
```